### PR TITLE
feat:Added Modal

### PR DESCRIPTION
**Pull request by ianuragsharma**
image

### CODE REVIEW

**Comment by som749898** [comment]
looks good to merge

File: styles/modal.css [hunk 1]
```diff
@@ -0,0 +1,23 @@
+.modal {
+  border: 2px solid var(--brand-secondary);
+  max-width: 22rem;
+  border-radius: 5px;
+  padding: 10px;
+  z-index: 1;
```
Comment (Naveenchandar) [inline styles/modal.css]: Z index won't be effective without position. Either define position or else you can remove this property.

**Review by aurobindoGupta — APPROVED** [review]
good work!!

File: styles/modal.css (hunk 1 above)
Comment (ianuragsharma) [reply styles/modal.css]: ok

**Review by Naveenchandar — APPROVED** [review]
Looks good

**Pull request merged**